docs/source/_static/versionwarning.js

```diff
@@ -30,6 +30,7 @@
             // Path of the page
             var location_array = location.pathname.split('/');
             var versionPath = location_array[2];
+            var versionNumber = Number(versionPath.match(/^\d+/))
```

Comment:
This is `majorVersionNumber` from the `major.minor.patch` format, right? A really minor nit I would probably rename it so it does not cause confusion.

Reply:
Yes, correct. It is the major part of the version format. Will rename 👍